src/geojson/geojson_intf.ml

```diff
@@ -204,10 +204,13 @@ module type S = sig
 
     val geometry : t -> Geometry.t option
     val properties : t -> json option
+    val foreign_members : t -> (string * json) list option
 
     include Json_conv with type t := t and type json := json
 
     val v : ?properties:json -> Geometry.t -> t
+
+    val w : ?foreign_members:(string * json) list -> Geometry.t -> t
```

Comment:
For now, seeing as a `Feature`, `Geometry` and `FeatureCollection` can all have `foreign_members` let's only add it to the `S.t` type (very similar to the bounding box implementation).